bin/cask

```diff
@@ -237,25 +237,30 @@ def find_best_emacs():
         return 'emacs'
 
 
+def get_emacs_from_env():
+    """Get the Emacs executable as specified by the environment.
+
+    Use the command given by the environment variable ``$CASK_EMACS`` if set.
+    Failing that use ``$EMACS``, unless the value of this variable does not
+    refer to a real Emacs executable (i.e. Cask is run from inside Emacs).
+
+    """
+    return ENVB.get(b'CASK_EMACS') or \
```

Comment:
Please use parenthesis around the whole expression.  Breaking lines with backslashes is discouraged in Python.